templates/vhost.conf

```diff
@@ -2,12 +2,18 @@
 
 <VirtualHost *:{{ port }}>
         ServerName {{ _website_domain }}
-    {% if _force_tls and port|int == 80 %}
+    {% if (_force_tls and port|int == 80) or reverse_proxy is defined %}
         RewriteEngine On
         # for lets encrypt automation
         RewriteCond %{REQUEST_URI} !^/.well-known/acme-challenge/
-        RewriteCond %{HTTPS} off
-        RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+
+        {% if _force_tls %}
+          RewriteCond %{HTTPS} off
+          RewriteRule (.*) https://%{HTTP_HOST}%{REQUEST_URI}
+        {% else if reverse_proxy is defined %}
+          RewriteRule ^/(.*)         {{ reverse_proxy }}$1 [L,P]
```

Comment:
what if reverse_proxy does not end with a slash? As $1 is not capturing the root slash I guess it would break.